## MESSAGE
# This message contains a connection between two rooms
# which are called "this_room" and "connected_to".
# The connectio is through a door.

# A room "this_room" is connected to
string connected_to

# The door that connects "this_room" and "connected_to"
string through_door
